test/src/engine/BMGameTest.php

```diff
@@ -1732,9 +1728,9 @@ public function test_update_game_state_end_turn() {
                                                    array($die2));
         $this->object->nRecentPasses = 0;
         $this->object->gameState = BMGameState::END_TURN;
+        $this->object->activePlayerIdx = 0;
         $this->object->update_game_state();
         $this->assertEquals(BMGameState::START_TURN, $this->object->gameState);
-        $this->assertTrue(isset($this->object->activeDieArrayArray));
```

Comment:
I guess this is just because I don't see any real value in maintaining these tests. If something is going to break, I'll see that anyway, and having this particular test break won't actually tell me more than something's broken.